assets/js/modules/adsense/index.js

```diff
@@ -45,10 +46,22 @@ import {
 	ERROR_CODE_ADBLOCKER_ACTIVE,
 } from './constants';
 import { WIDGET_AREA_STYLES } from '../../googlesitekit/widgets/datastore/constants';
-
 export { registerStore } from './datastore';
 
 export const registerModule = ( modules ) => {
+	// This is called inside `registerModule` to prevent this file from having
+	// side-effects. This is used to show notifications in the AdSense dashboard.
+	/**
+	 * Add components to the Notification requests.
+	 */
+	addFilter(
+		'googlesitekit.ModulesNotificationsRequest',
+		'googlesitekit.adsenseNotifications',
+		( notificationModules ) => {
+			return notificationModules.concat( 'adsense' );
```

Comment:
As the only instance of this filter, it makes sense to simplify this to use module slugs instead of the odd object structure it had before.